org.mwc.debrief.legacy/src/Debrief/ReaderWriter/Word/ImportNarrativeDocument.java

```diff
@@ -2184,55 +2376,72 @@ public void processThese(final ArrayList<String> strings) {
 						// clear the appended flag
 						appendedToPreviousCtr = 0;
 					}
-
-					// did we process anything?
+					final String type;
 					if (thisN.type != null) {
-						// ok, process the entry
-						switch (thisN.type) {
-						case "FCS": {
-							// add a narrative entry
-							addEntry(thisN);
-
-							// create track for this
-							try {
-								addFCS(thisN);
-							} catch (final StringIndexOutOfBoundsException e) {
-								// don't worry about panicking, it may not be an FCS after all
-							} catch (final NumberFormatException e) {
-								// don't worry about panicking, it may not be an FCS after all
-							}
-
-							// ok, take note that we've added something
-							dataAdded = true;
-
-							break;
-						}
-						default: {
-							// ok, just add a narrative entry for anything not recognised
-
-							// add a narrative entry
-							addEntry(thisN);
-
-							// ok, take note that we've added something
-							dataAdded = true;
-
-							break;
-
-						}
+						type = thisN.type;
+					} else if (thisN.platform != null) {
+						type = "None";
+					} else {
+						type = null;
+					}
+					if (type != null) {
+						if (typeVsCount.get(type) == null) {
+							typeVsCount.put(type, Integer.valueOf(1));
```

Comment:
@ayeshare - I'm pretty sure you can just used `1`, rather than `Integer.valueOf(1)`